.github/workflows/check-stylish-haskell.yml

```diff
@@ -3,6 +3,7 @@ name: Check Stylish Haskell
 on:
   merge_group:
   pull_request:
+  workflow_dispatch:
```

Comment:
do we need this?

Reply:
We don't need it, but I think it's good practice to be able to trigger the pipelines manually. It comes handy when the pipeline start failing and needs quick trial/error fixing.